app/routes.py

```diff
@@ -1,2 +1,317 @@
-from flask import Blueprint
+from app import db
+from app.models.video import Video
+from app.models.customer import Customer
+from app.models.rental import Rental
+from flask import request, Blueprint, make_response, jsonify
+import os
+import requests
+from datetime import datetime, timedelta
 
+now=datetime.now()
```

Comment:
Just note that `now` will be the time the application started.  So if the API keeps running for 3-5 days, your datetime here will become increasingly "outdated"  🤣 

In other words `now` won't be updated with each request, but set when you run `flask run`